app/models.py

```diff
@@ -0,0 +1,44 @@
+from app import db, login
+import datetime
+from hashlib import md5
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_login import UserMixin
+
+
+class Candidate(UserMixin, db.Model):
+  candidate_id = db.Column(db.String(32), primary_key=True)
+  username = db.Column(db.String(32), index=True, unique=True)
```

Comment:
Should we separate out "username", "email", and such out of the Candidate model? Otherwise, we can't really support multiple logins :'(

Reply:
Yeah I've been on the fence about this. I understand the desire to have logins that are not a candidate, but this way the workflow is:
- Create Account
- Log in
- Fill out form
- Done
Whereas otherwise it's
- Create candidate account
- Create user account
- Log in to user account
- Request access to candidate page
- Candidate accepts request
- User creates page on candidate's behalf
Also that means we will have to introduce access management tools for revoking access. Overall I think it's a lot more complexity than what we need for an MVP

Reply:
What if we went somewhere in the middle? We don't need all that complexity for the user in the MVP, but we could setup the DB now so that in the future we're capable of supporting it without too much data-transfer. 

So we supported multiple users from the database perspective, but there's only one user from the flask and frontend perspective.